test/plugins/sensor_demo/custom_sensor/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+
+project(custom_rendering_sensor)
+
```

Comment:
Since this CMakeLists.txt is being included by the top level, I don't think these lines are necessary.

Reply:
removed cmake version line.